src/lib/zombienetInfo.ts

```diff
@@ -0,0 +1,57 @@
+export type zombienetInfo = typeof zombienet;
+
+export type BinaryNames = "zombienet" | "polkadot" | "astar-collator";
+export const zombienet = {
+  version: "1.3.89",
+  downloadUrl: {
+    darwin: {
```

Comment:
Polkadot binaries cannot be executed directly in MacOS, so we can remove them and ignore bin downloads for darwin performing a small OS check. Also add a warning disclaimer in `zombienet init` command for MacOS users. This can be similar to the one in [zombienet documentation](https://paritytech.github.io/zombienet/cli/setup.html) (bottom of the page). For the OS check anticipate a common helper that will be shared with this [task](https://github.com/inkdevhub/swanky-cli/blob/0914aed04d0239be6c9b377001f07c26b28b7814/src/commands/check/index.ts#L57-L71) in #205